packages/geo/src/coordinates/latlon/internal/index.ts

```diff
@@ -0,0 +1,141 @@
+// __private-exports
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { Patterning } from '@/patterning';
+
+export type Errors = string[];
+
+export type Lat = 'LAT';
+export type Lon = 'LON';
+export type Format = `${Lat}${Lon}` | `${Lon}${Lat}`;
+
+export enum Compass {
+  North = 'N',
+  South = 'S',
+  East = 'E',
+  West = 'W',
+}
+
+export enum Limits {
+  Lat = 90,
+  Lon = 180,
+}
+
+export enum Symbols {
+  Degrees = '°',
+  Minutes = "'",
+  Seconds = '"',
+  Separator = '/',
+}
+
+export const BEARINGS = {
+  // [positive, negative] so that "isNegative" can be used to index into the array
+  LAT: [Compass.North, Compass.South],
+  LON: [Compass.East, Compass.West],
+} as const;
+
+export const DEFAULT_FORMAT: Format = 'LONLAT';
+
+export const LIMITS: Record<Format, [Limits, Limits]> = {
+  LATLON: [Limits.Lat, Limits.Lon],
+  LONLAT: [Limits.Lon, Limits.Lat],
+};
+
+export const SYMBOL_PATTERNS = {
+  LAT: new RegExp(`[${BEARINGS.LAT.join('')}]`),
```

Comment:
A fun thing we did at the very start of this endeavor for the sake of other devs was added in https://regex-vis.com/ links as comments above the regex so folks could see an explanation of it. Might be a good idea to pop those in as well during the polish stage. E.g. https://regex-vis.com/?r=%2F%5Cs*%2F

Reply:
Another site with a slightly different visual style: https://www.nodexr.net/?parse=(%5B-%2B%5D)%5Cs*

Reply:
Oh! I like those. ( `{@link https://www.nodexr.net/?parse=(%5B-%2B%5D)%5Cs }`)